odl/test/tomo/backends/astra_setup_test.py

```diff
@@ -340,6 +346,16 @@ def test_astra_algorithm():
                                           proj_id, impl)
         astra.algorithm.delete(alg_id)
 
+
+def test_astra_algorithm_gpu():
+    """Create ASTRA algorithm object on GPU."""
+
```

Comment:
The whole point of this PR was to safe-guard for non-cuda users. But apparently I forgot to add
```python
if not odl.tomo.ASTRA_CUDA_AVAILABLE:
    pytest.skip("ASTRA cuda not available")
```
at this point...

Reply:
Now it is updated on the line below